ui/apps/platform/src/Containers/Clusters/ClusterDeployment.tsx

```diff
@@ -55,10 +55,14 @@ function ClusterDeployment({
                         spaceItems={{ default: 'spaceItemsMd' }}
                     >
                         <Title headingLevel="h3">1. Download files</Title>
-                        <Text>Download the required configuration files, keys, and scripts.</Text>
+                        <Text>
+                            Download the required configuration files, keys, and scripts. Enabling
+                            automatic upgrades creates a powerful service account in your secured
+                            cluster that will be used to perform the upgrades.
+                        </Text>
                         <Switch
-                            label="Cluster is configured to allow future automatic upgrades"
-                            labelOff="Click to configure cluster to allow future automatic upgrades"
+                            label="Automatic upgrades ON: Will upgrade secured clusters with versions behind central's."
```

Comment:
How about
```suggestion
                            label="Automatic upgrades ON: clusters are upgraded automatically when central is updated"
```

Reply:
That would be less correct. Let's say I have automatic upgrades turned on and my secured clusters behind central version. If I toggle automatic upgrades on it will upgrade immediately, whereas your change suggests that it would upgrade only when central upgrades.

How about "Automatic upgrades ON: Secured clusters are upgraded automatically when behind Central in version."

Reply:
What happens if the secured cluster is ahead of central and we have the auto upgrader on? I'm not too familiar with the code but I think the auto upgrader just matches whatever version central is running. So, if that's the case, maybe `Automatic upgrades ON: Secured clusters are upgraded automatically to match Central`?

Reply:
When central is behind sensor the automatic upgrade toggle will not downgrade sensor. You can still trigger a downgrade for sensor to match central by clicking the upgrade button though.

Reply:
> You can still trigger a downgrade for sensor to match central by clicking the upgrade button

Gotcha! 👍

Reply:
I'm having trouble with the use of  **behind** in that context. I think it is not clear enough for customers to understand what it is doing (I fear that's mostly a German language construct that's leading to this).
So how about 
`Secured Clusters are automatically upgraded to match Central version` ?

Reply:
> Secured Clusters are upgraded automatically to match Central version

That is not strictly the case. Secured clusters are upgraded to match the central version iff sensor version < central version. In the case of sensor version > central version having the toggle on would do nothing, but there would be a button to manually "upgrade" in which case sensor would downgrade to match central.

With a version *behind* central, a version *lower than* central's is what I mean. Would it be better if I formulate it that way?

Reply:
I like `Secured Clusters are upgraded automatically to match Central version` more. I think it's cleaner. The key here is the word `upgrade` which normally is understood as getting a new version. So in this case this reads as sensor upgrades if sensor version < central version.

If we want to be very precise and explain the whole *automatically* upgrade only is sensor version < central version and you can *manually* downgrade if sensor version > central version, I think the docs are a better place for that. 

I will not block the PR if we go either way though 🙂

Reply:
Well since both you and Matthias prefer it this way I'll go with the democratic approach 👍